assets/cluster-bootstrap/route-controller-ns.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
```

Comment:
Please rename this file to something like 00-route-controller-ns.yaml so it gets applied before the rolebinding

Reply:
Renamed, thank you!